packages/docusaurus-theme-search/src/theme/SearchBar/index.module.css

```diff
@@ -0,0 +1,703 @@
+  background-color: var(--markprompt-background);
+  border-radius: var(--markprompt-radius);
+  border: 1px solid var(--markprompt-border);
+  box-shadow: hsl(206 22% 7% / 35%) 0px 10px 38px -10px,
+    hsl(206 22% 7% / 20%) 0px 10px 20px -15px;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 80vw;
+  max-width: 600px;
+  height: calc(100vh - 200px);
+  max-height: 600px;
+  animation-name: contentShow;
+  animation-duration: 300ms;
+  animation-fill-mode: both;
+  transition-timing-function: cubic-bezier(0.25, 0.4, 0.55, 1.4);
+  color: var(--markprompt-foreground);
+  overflow: hidden;
+  display: grid;
+  grid-template-rows: auto 1fr;
+}
+
+.MarkpromptClose {
+  all: unset;
+  position: absolute;
+  top: 0.75rem;
+  right: 0.75rem;
+  color: var(--markprompt-foreground);
+  width: var(--markprompt-button-icon-size);
+  height: var(--markprompt-button-icon-size);
+  padding: 0.25rem;
+  border-radius: 4px;
+  cursor: pointer;
+  display: grid;
+  place-items: center;
+  transition-property: box-shadow;
+  transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
+  transition-duration: 150ms;
+}
+
+.MarkpromptClose:hover {
+  opacity: 0.8;
+}
+
+.MarkpromptClose:focus {
+  box-shadow: inset 0 0 0 2px var(--markprompt-primary);
+}
+
+.MarkpromptSearchIcon {
+  position: absolute;
+  top: 1rem;
+  left: 1.25rem;
+  color: var(--markprompt-foreground);
+  width: var(--markprompt-button-icon-size);
+  height: var(--markprompt-button-icon-size);
+  cursor: pointer;
+  display: grid;
+  place-items: center;
+}
+
+.MarkpromptTitle {
+  margin-block-start: 0;
+}
+
+.MarkpromptPrompt {
+  border-left: none !important;
+  border-right: none !important;
+  border-top: none !important;
+  outline: none !important;
+  border-bottom: 1px solid var(--markprompt-border);
+  box-shadow: none;
+  width: 100%;
+  padding-left: 3.5rem;
+  padding-right: 3.5rem;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+  font-size: var(--markprompt-text-size);
+  background-color: var(--markprompt-background);
+  color: var(--markprompt-foreground);
+  caret-color: var(--markprompt-primary);
+}
+
+.MarkpromptPrompt:focus {
+  outline: 2px solid transparent;
+  outline-offset: 2px;
+  box-shadow: none;
+}
+
+.MarkpromptPrompt::placeholder {
+  color: var(--markprompt-mutedForeground);
+}
+
+.MarkpromptPrompt:focus {
+  outline: 2px solid var(--markprompt-mutedForeground);
+}
+
+.MarkpromptReferences {
+  background-color: var(--markprompt-muted);
+  color: var(--markprompt-mutedForeground);
+  border-top: 1px solid var(--markprompt-border);
+}
+
+.MarkpromptAnswer {
+  overflow-y: auto;
+  overflow-x: hidden;
+  height: 100%;
+  padding: 1rem 2rem;
+  scroll-behavior: smooth;
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+  font-size: var(--markprompt-text-size);
+}
+.MarkpromptAnswer::-webkit-scrollbar {
+  display: none;
+}
+.MarkpromptAnswer :not(:last-child) .caret {
+  display: none;
+}
+.MarkpromptAnswer {
```

Comment:
I saw this was taken from `with-css-modules`, but I think it would be preferable to add a `data-markprompt-answer` attribute to a wrapper around the current Answer component return value and allow users to add their own styles rather than include all of this bloat. This is really tricky to override in userland if folks would like to do so.